account_analytic_required/models/account.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2011-2020 Akretion - Alexis de Lattre
+# Copyright 2016-2020 Camptocamp SA
+# Copyright 2020 Druidoo - Iván Todorovich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, exceptions, fields, models
+
+
+class AccountAccount(models.Model):
+    _inherit = "account.account"
+
+    analytic_policy = fields.Selection(
+        selection=[
+            ("optional", "Optional"),
+            ("always", "Always"),
+            ("posted", "Posted moves"),
+            ("never", "Never"),
+        ],
+        string="Policy for analytic account",
+        default="optional",
+        company_dependent=True,
```

Comment:
Could you please port https://github.com/OCA/account-analytic/pull/692 to this current PR?